OBSRemoteParameters_StreamlabsParameter.py

```diff
@@ -7,12 +7,14 @@
 
 Version
 
+	1.4.0
+		Added $OBSsetVolume set volume for given source, 1-100
 	1.3.0
 		$OBStimedSource now has an onoff or offon mode and $OBSscene now can accept an optional delay.
 		All parameters now use threading in Python to execute actions.
 	1.2.0
 		Added $OBStimedScene swap to a given scene for a set period of time, then swap to another given scene.
-		Changed script name to _StreamlabsSystem to accomedate the change in the bot name.
+		Changed script name to _StreamlabsParameter to accomedate the change in the bot name.

```

Comment:
Seems like the script is named _StreamlabsParameter.py ? Not sure if this was a typo